src/box/lua/config/descriptions.lua

```diff
@@ -0,0 +1,2530 @@
+    `election_fencing_mode` to off.
+]])
+
+I['replication.election_mode'] = format_text([[
+    A role of a replica set node in the leader election process.
+
+    The possible values are:
+
+    - `off`: a node doesn't participate in the election activities.
+    - `voter`: a node can participate in the election process but can't be
+      a leader.
+    - `candidate`: a node should be able to become a leader.
+    - `manual`: allow to control which instance is the leader explicitly
+      instead of relying on automated leader election. By default, the instance
+      acts like a voter - it is read-only and may vote for other candidate
+      instances. Once `box.ctl.promote()` is called, the instance becomes a
+      candidate and starts a new election round. If the instance wins the
+      elections, it becomes a leader but won't participate in any new elections.
+]])
+
+I['replication.election_timeout'] = format_text([[
+    Specifies the timeout (in seconds) between election rounds in the leader
+    election process if the previous round ended up with a split vote.
+
+    It is quite big, and for most of the cases, it can be lowered to 300-400 ms.
+
+    To avoid the split vote repeat, the timeout is randomized on each node
+    during every new election, from 100% to 110% of the original timeout value.
+    For example, if the timeout is 300 ms and there are 3 nodes started the
+    election simultaneously in the same term, they can set their election
+    timeouts to 300, 310, and 320 respectively, or to 305, 302, and 324, and
+    so on. In that way, the votes will never be split because the election on
+    different nodes won't be restarted simultaneously.
+]])
+
+I['replication.failover'] = format_text([[
+    A failover mode used to take over a master role when the current master
+    instance fails. The following modes are available:
+
+    - `off`: Leadership in a replica set is controlled using the
+      `database.mode` option. In this case, you can set the `database.mode`
+      option to rw on all instances in a replica set to make a master-master
+      configuration.
+    - `manual`: Leadership in a replica set is controlled using the
+      `<replicaset_name>.leader` option. In this case, a master-master
+      configuration is forbidden.
+    - `election`: Automated leader election is used to control leadership in a
+      replica set. In the election mode, `database.mode` and
+      `<replicaset_name>.leader` shouldn't be set explicitly.
+    - `supervised`: (Enterprise Edition only) Leadership in a replica set is
+      controlled using an external failover coordinator.
+
+    Notes:
+
+    The default database.mode is determined as follows: `rw` if there is one
+    instance in a replica set; `ro` if there are several instances.
+
+    In the`manual` mode, the `database.mode` option cannot be set explicitly.
+    The leader is configured in the read-write mode, all the other instances
+    are read-only.
```

Comment:
Missed whitespace after the first 'the'.

Reply:
Fixed. See above.